osrm.go

```diff
@@ -30,6 +30,29 @@ type Config struct {
 	Client HTTPClient
 }
 
+// ResponseStatus represent OSRM API response
+type ResponseStatus struct {
+	Code        string `json:"code"`
+	Message     string `json:"message"`
+	DataVersion string `json:"data_version"`
```

Comment:
Is it optional or required here?

Reply:
OSRM doc says it's optional.
I understand your question, but not sure we have to make the field as a pointer.
WDYT?

Reply:
I just not very fluent with json parsing in go — would this fail if there is no `data_version` field?

Reply:
No, it will be an empty string in the case.